app/src/app/home/pages/element/results.component.html

```diff
@@ -1,4 +1,4 @@
-<section class="wsl-page-tab" id="wsl-element-tab-results">
+<section class="wsl-page-tab" [id]="tabId">
```

Comment:
Unrelated fix: This html page is now used by assumptions component in addition to the results component so element page always has two copies of this component so we should not be using the same id element. This resulted in an error being thrown when user attempted to expand the content of a complex key via pressing `v` because the webapp incorrectly calculated the total number of available rows on the active list.